rnmodules/react-native-kb/android/src/main/java/com/reactnativekb/KbModule.kt

```diff
@@ -109,14 +109,8 @@ internal class KbModule(reactContext: ReactApplicationContext?) : KbSpec(reactCo
         return GuiConfig.getInstance(reactContext.getFilesDir())?.asString()
     }
 
-
-    // only old arch, uncomment
-    // override fun getConstants(): MutableMap<String, Any>? {
-    //     return getTypedExportedConstants()
-    // }
-
-    // newarch @Override
-    override fun getTypedExportedConstants(): MutableMap<String, Any> {
+    @ReactMethod(isBlockingSynchronousMethod = true)
```

Comment:
how the built in `getConstants` works is different between old and new arch and thats annoying for no reason. instead we just make a new function without any baggage